sdk-overrides/include/c_types.h

```diff
@@ -0,0 +1,2 @@
+#define ICACHE_FLASH
+#include_next "c_types.h"
```

Comment:
Objection, your honour! :)
When I hit that other shiny green button just before, our `c_types.h` got decimated and should stay that way.

Reply:
Yup, agreed. All part of the rebaseline TBD today. The reason for the wrapper is that ICACHE_FLASH was only used in this include.